remill/Arch/AArch64/Arch.cpp

```diff
@@ -729,6 +729,14 @@ static void AddPostIndexMemOp(Instruction &inst, Action action,
 }
 
 static bool MostSignificantSetBit(uint64_t val, uint64_t *highest_out) {
+  #if defined(__GCC__) || defined(__clang__)
```

Comment:
Can you bring the `#if` to have no indentation, and use `__has_builtin(__builtin_clzll)`. For example: https://godbolt.org/g/txSAup